lib/rubocop/cop/lint/assignment_in_condition.rb

```diff
@@ -24,7 +24,14 @@ module Lint
       class AssignmentInCondition < Cop
         include SafeAssignment
 
-        MSG = 'Assignment in condition - you probably meant to use `==`.'.freeze
+        MSG_WITH_SAFE_ASSIGNMENT_ALLOWED = [
```

Comment:
Please move these into multiline string literals, to be on par with the rest of the project. We generally use `\` to do so. 🙂 

```ruby
MSG =
  "More than one " \
  "line."
```